### PR TITLE
[WIP] Vrouter env variables with user defined values

### PR DESCRIPTION
**Pull request by Janek3d**
Add option for user to define values of vrouter env variables.

### CODE REVIEW

File: pkg/apis/contrail/v1alpha1/vrouter_types.go [hunk 1]
```diff
@@ -50,8 +50,9 @@ type VrouterStatus struct {
 // VrouterSpec is the Spec for the vrouter API.
 // +k8s:openapi-gen=true
 type VrouterSpec struct {
-	CommonConfiguration  PodConfiguration            `json:"commonConfiguration,omitempty"`
-	ServiceConfiguration VrouterServiceConfiguration `json:"serviceConfiguration"`
+	CommonConfiguration       PodConfiguration            `json:"commonConfiguration,omitempty"`
+	ServiceConfiguration      VrouterServiceConfiguration `json:"serviceConfiguration"`
+	EnvVariablesConfiguration map[string]string           `json:"envVariablesConfiguration,omitempty"`
```
Comment (Dysproz) [inline pkg/apis/contrail/v1alpha1/vrouter_types.go:55]: Should be inside ServiceConfiguration

**Review by Dysproz — CHANGES_REQUESTED** [review]

File: pkg/apis/contrail/v1alpha1/vrouter_types.go [hunk 2]
```diff
@@ -50,32 +50,33 @@ type VrouterStatus struct {
...
+	CommonConfiguration  PodConfiguration            `json:"commonConfiguration,omitempty"`
+	ServiceConfiguration VrouterServiceConfiguration `json:"serviceConfiguration"`
 }
 
 // VrouterServiceConfiguration defines all vRouter service configuration
 // +k8s:openapi-gen=true
 type VrouterServiceConfiguration struct {
 	VrouterConfiguration      `json:",inline"`
 	VrouterNodesConfiguration `json:",inline"`
+	EnvVariablesConfiguration map[string]string `json:"envVariablesConfiguration,omitempty"`
```
Comment (psykulsk) [inline pkg/apis/contrail/v1alpha1/vrouter_types.go:62]: You added it to VrouterConfiguration struct already, so you do not need to add it here again

**Review by psykulsk — CHANGES_REQUESTED** [review]

**Comment by Dysproz** [comment]
recheck

**Comment by psykulsk** [comment]
recheck

**Comment by psykulsk** [comment]
recheck ha

**Pull request closed without merging**